modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -189,7 +189,11 @@ def shared_validation
           # Custom validations for 526 submission, we must check this first
           @claims_api_forms_validation_errors = validate_form_526_submission_values!(target_veteran)
           # JSON validations for 526 submission, will combine with previously captured errors and raise
-          validate_json_schema
+          if params[:action] == 'synchronous'
+            validate_json_schema('526_SYNCHRONOUS')
+          else
+            validate_json_schema
+          end
           # if we get here there were only validations file errors
```

Comment:
This adjustment was a part of my concern, since it seems _only_ required to get rswag to run correctly